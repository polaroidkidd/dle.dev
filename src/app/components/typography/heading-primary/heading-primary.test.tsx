import React from "react";
import renderer from "react-test-renderer";
import { HeadingPrimary } from "./index";

test("Render HeadingPrimary with text", () => {
  const tree = renderer.create(<HeadingPrimary text="test heading"/>).toJSON();
  expect(tree).toMatchSnapshot();
});

test("Render HeadingPrimary with empty string", () => {
  const tree = renderer.create(<HeadingPrimary text=""/>).toJSON();
  expect(tree).toMatchSnapshot();
});
