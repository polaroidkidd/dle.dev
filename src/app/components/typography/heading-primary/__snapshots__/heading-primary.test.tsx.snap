// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render HeadingPrimary with empty string 1`] = `
<h1
  className="heading-primary"
>
  
</h1>
`;

exports[`Render HeadingPrimary with text 1`] = `
<h1
  className="heading-primary"
>
  test heading
</h1>
`;
